### ISSUE TITLE
Benchmark IDD dataset

### ISSUE THREAD

**Issue by ghost** [prajjwal1/autonomous-object-detection#6]
I am trying to replicate the baseline results on IDD dataset with Faster-RCNN. For this baseline, did you use objects of 'small' area (as per COCO standards) for training and evaluation? Also, can you upload the trained Faster R-CNN baseline model for IDD dataset? I see only bdd100k_24.pth and bdd_idd_incremental.pth uploaded.

Finally, I like your idea of using incremental learning. However, in section 5.3 (Gradual unfreezing) of your paper, the first line states "Training the entire model on a different domain at once leads to catastrophic forgetting". Based on how your model suffered catastrophic forgetting due to domain shift from BDD to IDD dataset and the fact that PyTorch Faster RCNN model has been trained on COCO dataset, did you observe similar catastrophic forgetting in your model while achieving Faster RCNN baseline results on IDD dataset?

Lastly, how can one decide if a model is suffering from catastrophic forgetting due to domain shift? What techniques did you use for the same?

**Comment by prajjwal1** [comment]
1.  I used all ['all' ,'large', 'medium']. The table in the paper makes this clear.
2. I didn't train solely on IDD. I trained first on BDD100k and then on IDD with incremental learning approaches mentioned in the paper. So that's the reason why I didn't upload.
3. Yes, that would definitely occur. Even COCO is very different from BDD or any other autonoumous navigation datasets. I needed a good starting point for initialization, so I chose pretrained weights from COCO. I also used some other initialization methods such as LSUV, but didn't get any comparable performance as I received from using pretrained COCO weights.
4. You can easily see this. First train on domain A, train on domain B with a different linear layer, and then perform inference on domain A with the same linear layers used when you performed training on domain A. You can easily see the decrement in performance/accuracy. All the techniques I used to cope up with this problem can be found in the paper and `exp` directory.

**Comment by ghost** [comment]
Thanks for the prompt reply.
